### PR TITLE
Milestone 10: Authentication with Devise gem

### PR DESCRIPTION
**Pull request by SaidRasinlic**
### In this milestone I've done the following things:

Required Changes ♻️ (uncomment confirmable in Devise) : Resolved removing commented confirmable lines of code ✔️

- Made sure that authentication is added for Sign In and Sign Up for an user.
-  Added look & feel for Views.
- Refactored code to use Rails best practices.

### CODE REVIEW

File: db/migrate/20220805151703_add_devise_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
```
Comment (elisha2kyakpo1) [inline db/migrate/20220805151703_add_devise_to_users.rb:25]: [ ] Kindly uncomment this part, remember we have confirmable in our user's model and we want user's to confirm their registration by email.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @SaidRasinlic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The PR title and description looks professional✔️
- Descriptive commit messages✔️
- No linter errors✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220805151703_add_devise_to_users.rb (hunk 1 above)
Comment (SaidRasinlic) [reply db/migrate/20220805151703_add_devise_to_users.rb]: **Successfully uncommented confirmable for User's model ✔️.**

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 🍷 

## Hi @SaidRasinlic 👋, 

Well done implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**